code/modules/reagents/reagent_containers/food/cans.dm

```diff
@@ -24,6 +29,159 @@
 		return TRUE
 	. = ..()
 
+/obj/item/reagent_containers/food/drinks/cans/update_icon()
+	cut_overlays()
+	if(fuselength)
+		var/image/fuseoverlay = image('icons/obj/fuses.dmi', icon_state = "fuse_short")
+		switch(fuselength)
+			if(6 to INFINITY)
+				fuseoverlay.icon_state = "fuse_long"
+		fuseoverlay.pixel_x = can_size_overrides["x"]
+		fuseoverlay.pixel_y = can_size_overrides["y"]
+		add_overlay(fuseoverlay)
+
+/obj/item/reagent_containers/food/drinks/cans/attackby(obj/item/W, mob/user)
+	if(istype(W, /obj/item/steelwool))
+		if(is_open_container())
+			switch(fuselength)
+				if(0 to 9)
```

Comment:
You reuse various fuselength numbers a lot. Maybe make them defines.

Reply:
I added a few more defines in the most recent commit - I won't mark this as resolved in case you think there's more changes I can make, though I think I've stretched it as far as I can.